api/serializers.py

```diff
@@ -21,12 +23,21 @@ class Meta:
             'payload_request', 
             'related_ioc',
         )
-    
+
+class EnrichmentSerializer(IOCSerializer):
```

Comment:
why are u extending IoCserializer here ? use normal Serializer (`serializers.Serializer`) and specify the `fields` variable in `Meta` class.